configure

```diff
@@ -13426,7 +13477,7 @@ find_lib()
         host_lib_path="$ac_default_prefix/$base_libdir $ac_default_prefix/$base_bindir /usr/$base_libdir /usr/local/$base_libdir"
     fi
     for path in $env_lib_path $gcc_bin_path $gcc_lib_path $host_lib_path; do
-        lib=`ls -- $path/$1 2>/dev/null | sed -e '/\.so\..*\./d' -e 's,.*/,,' | sort | tail -1`
+        lib=`ls -- $path/$1 2>/dev/null | sed -e 's,.*/,,' | $AWK '{print length() " " $0;}' | sort -n -r | tail -1 | sed 's/^[0-9]\+ //'`
```

Comment:
Does this work on Macos? Because the output of configure gives an error on the next line.

Reply:
Possibly something to do with `awk` on macOS, and possibly on other BSDs by extension?

Is this written with `gawk` in mind? (No, I have no awk skills at all.)